AI/Nullkiller/AIGateway.cpp

```diff
@@ -92,8 +92,8 @@ void AIGateway::heroMoved(const TryMoveHero & details, bool verbose)
 	validateObject(details.id); //enemy hero may have left visible area
 	auto hero = cb->getHero(details.id);
 
-	const int3 from = CGHeroInstance::convertPosition(details.start, false);
-	const int3 to = CGHeroInstance::convertPosition(details.end, false);
+	const int3 from = hero->convertToVisitablePos(details.start);
+	const int3 to = hero->convertToVisitablePos(details.end);
```

Comment:
Thanks for noticing.
Yes, it may be null. Will add workaround.
Will do proper fix later, in a separate PR. 
There are many edge cases when applying this netpack, don't want to increase this PR in size